ExampleSensor/README.md

```diff
@@ -0,0 +1,29 @@
+# Example Sensor
+
+This is an example `README` for the sensor "Example Sensor". It should include the following content:
+
+- Links to sensor documentation/references;
+- Testing setup, execution, and expected output;
+- A list of dependencies (and links to those);
+- Everything else that is listed under [the master `README`](../README.md).
+
```

Comment:
Let's have the testing setup (pictures, etc,) documented on google drive and refer to it here using view-only links.

Reply:
Done!